### ISSUE TITLE
RRE-Finder in AntiSmash

### ISSUE THREAD

**Issue by tamuanand** [antismash/antismash#296]
Hi 

I have a question on RRE-Finder integration with antiSMASH. I believe @Alexamk has some PR in place.

[RRE-Finder: a Genome-Mining Tool for Class-Independent RiPP Discovery](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7470986/#s3title)   states
```
Integration of RRE-Finder into RODEO and antiSMASH.
Precision mode models have also been incorporated into both the GitHub and Web tool versions of RODEO 2 (http://rodeo.scs.illinois.edu). Included is an option to score RRE domains, which, if selected, will show which precision-mode models are matched, along with the default Pfam matches. The integration of precision mode is in progress for version 6.0 of antiSMASH, which is currently in the development phase and will be reported elsewhere
```

Any projected timelines for RRE-FINDER in antiSMASH and also the next antiSMASH (will it be antiSMASH 5.2 or antiSMASH 6.0) 

Thanks in advance.

**Comment by Alexamk** [comment]
Hello,
RREFinder precision mode integration is planned for antiSMASH 6.0.
Best,
Alexander

**Comment by tamuanand** [comment]
Thanks @Alexamk - Any idea when antiSMASH 6.0 might be available?
> Hello,
> RREFinder precision mode integration is planned for antiSMASH 6.0.
> Best,
> Alexander

**Comment by SJShaw** [comment]
If everything goes as planned, 5.2 is likely to be some time in the next few weeks and 6.0 will be around two months later.

**Comment by tamuanand** [comment]
Thanks @SJShaw - eagerly looking forward to 6.0

I am assuming pfam2go will also be updated with 5.3 as discussed at https://github.com/antismash/antismash/issues/257#issuecomment-613869987

**Comment by SJShaw** [comment]
That will be in 5.2 as it's already merged.,Anything merged after 5.2 will be in 6.0 (or possibly a 5.2.x, if necessary).

**Comment by shravansukumar123** [comment]
Hi! I'm looking forward to using these features- Do you have an estimate for Antismash 6.0 release? Thanks!
